config.rb

```diff
@@ -90,6 +90,8 @@ def self.test?
   override :managed_service, false, bool
   override :sanctioned_countries, "CU,IR,KP,SY", array(string)
   override :hetzner_ssh_key, string
```

Comment:
Was wondering if i should rename this to `hetzner_ssh_public_key`

Reply:
I think that makes sense. Also, it would make sense to put that to a separate commit, so that the "boring" refactoring code wouldn't be intertwined with more interesting logic changes.

Reply:
Done